pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+Provide a brief description of what this PR does, and why it's needed.
+
+### Does this PR relate to an RChain JIRA issue? 
+If applicable, add link to corresponding JIRA issue.
+
+### Complete this checklist before you submit the PR
+- [ ] This PR contains no more than 200 lines of code.
```

Comment:
Maybe some like `This PR contains no more than 200 lines of code or is irreducible.`

Reply:
Thanks, @KentShikama. The goal here is to assure PRs represent concise changes to facilitate the review process. Does offering the `is irreducible` option then open the door for a PR that is larger? And, if so, is there a scenario where this would be reasonable?

Reply:
The classic example here is something that involves a lot of reformatting or simple find/replace. For example something like https://github.com/rchain/rchain/pull/373 . The idea here is that it is conceptually simple but it touches a lot of lines of code. Maybe we can keep it as "This PR contains no more than 200 lines of code" and just ignore it when we absolutely have to.

Reply:
Or at least "This PR contains no more than 200 lines of code excluding test code"